htmlhint/README.md

```diff
@@ -1,2 +1,53 @@
 # vscode-htmlhint
-Integrates [htmlhint](https://github.com/yaniswang/HTMLHint) into VS Code.
\ No newline at end of file
+
+Integrates the [HTMLHnt](https://github.com/yaniswang/HTMLHint) static analysis tool into Visual Studio Code.
+
+![hero](hero.png)
+
+## Configuration
+
+The HTMLHint extension uses a bundled version of HTMLHint so no further configuration is required besides installing the extension.
+
+## Usage
+
+The HTMLHint extension will run HTMLHint on your open HTML files and report the number of errors on the Status Bar with details in the Problems panel (**View** > **Problems**).
+
+![status bar](status-bar.png)
+
+Errors in HTML files are highlighted with squiggles and you can hover over the squiggles to see the error message.
+
+![hover](hover.png)
```

Comment:
Nit on the images, but can you put them in a subfolder (e.g., `doc-images` or something appropriate?).